typescript/db/schema.prisma

```diff
@@ -95,15 +95,16 @@ model Dataset {
 model Workspace {
   // Id exists for regularity reasons.
   // It's more appropriate to query Workspaces using the slug
-  id               String        @id @default(uuid()) @db.Uuid
-  createdAt        DateTime      @default(now())
-  updatedAt        DateTime      @updatedAt
+  id               String          @id @default(uuid()) @db.Uuid
+  createdAt        DateTime        @default(now())
+  updatedAt        DateTime        @updatedAt
   deletedAt        DateTime?
-  name             String        @unique
-  slug             String        @unique
+  name             String          @unique
+  slug             String          @unique
   image            String?
   plan             WorkspacePlan
   stripeCustomerId String?
+  status           WorkspaceStatus @default(Active)
```

Comment:
Well, I've put this value here because of the existing workspaces in db (and it's required if I remember the error message correctly), which all are `Community` ones. I don't see why we should put those workspaces in trialing mode